lib/rialto/etl/writers/sparql_writer.rb

```diff
@@ -111,11 +113,6 @@ def post(statements)
           raise
         end
 
-        # Get the logger from the settings, or default to an effectively null logger
```

Comment:
Different approach from the SPARQL Statement Writer?

Reply:
Can you clarify? I don't see any logging in the sparql statement writer.

Reply:
Outdated comment, methinks.